### DISCUSSION TITLE
How to run project?

### DISCUSSION THREAD

**Discussion by bahodurnazarov** [KostyaEsmukov/smtp_to_telegram#42] [General]
Hello, I just started learn GO and don't know anything about docker. Please can you explain me how to run this project!
I really need this project.
Thank you!

**Comment by KostyaEsmukov** [comment]
The instructions for running this project are available in the readme: https://github.com/KostyaEsmukov/smtp_to_telegram#getting-started

Learning Golang is not required to just use the project, unless you want to make some changes. You may start with following the aforementioned readme.

Docker is a means of distribution of software, see https://docs.docker.com/get-docker/

**Reply by bahodurnazarov** [reply]
Hello,
![image](https://user-images.githubusercontent.com/62015918/202677876-ebcfe602-1865-4577-9e9e-bc38740d9ece.png)
Is it already running? 

how I can test it? 
Where should I put my email? 
Please explain, Thank you!

**Reply by bahodurnazarov** [reply]
I'm receiving message on outlook, I want to get it on telegram automaticly

**Reply by bahodurnazarov** [reply]
Mr Kostya, if you have time, Please reply, I'm waiting your response.
